go_live.md

```diff
@@ -2,7 +2,6 @@
 
 ## DNS, SSL & SMTP
 
-* Go to https://www.cloudflare.com/ and add your site, if not already done
 * Set up your DNS records (e.g CNAMES to heroku, TXT records for SparkPost, ...)
```

Comment:
do we have to decide, where to do that? i see you deleted cloudflare. is it, because there are too many different cases? since we have the chapter `cloudflare.md` for rails, it would maybe make sense to link it, since in most cases you have to change it there?